diff --git a/ltw/src/main/tinywrapper/main.c b/ltw/src/main/tinywrapper/main.c
index 3d1cdd6..2fea34b 100644
--- a/ltw/src/main/tinywrapper/main.c
+++ b/ltw/src/main/tinywrapper/main.c
@@ -307,9 +307,9 @@ const GLubyte* glGetString(GLenum name) {
     if(!current_context) return NULL;
     switch(name) {
         case GL_VERSION:
-            return (const GLubyte*)"3.3 OpenLTW (Built on: "__DATE__"/"__TIME__")";
+            return (const GLubyte*)"3.3 LTW §f(§dYukari Launcher§f)";
         case GL_SHADING_LANGUAGE_VERSION:
-            return (const GLubyte*)"4.60 LTW";
+            return (const GLubyte*)"4.60 LTW §f(§dYukari Launcher§f)";
         case GL_VENDOR:
             return (const GLubyte*)"artDev, SerpentSpirale, CADIndie";
         case GL_EXTENSIONS:
