diff --git a/MobileGlues-cpp/gl/getter.cpp b/MobileGlues-cpp/gl/getter.cpp
index dd476c7..cd06559 100644
--- a/MobileGlues-cpp/gl/getter.cpp
+++ b/MobileGlues-cpp/gl/getter.cpp
@@ -374,7 +374,7 @@ const GLubyte* glGetString(GLenum name) {
                     shadingLangString += GenerateRandomString(junkOpts);
                 }
             } else {
-                shadingLangString = baseVer + " MobileGlues with glslang and SPIRV-Cross";
+                shadingLangString = baseVer + " MobileGlues §f(§dYukari Launcher§f)§r";
             }
         }
 
diff --git a/MobileGlues-cpp/version.h b/MobileGlues-cpp/version.h
index dd075e5..4e809ac 100644
--- a/MobileGlues-cpp/version.h
+++ b/MobileGlues-cpp/version.h
@@ -24,7 +24,7 @@
 #define VERSION_RC_NUMBER 2
 #endif
 
-#define VERSION_SUFFIX ""
+#define VERSION_SUFFIX " §f(§dYukari Launcher§f)"
 
 #define MOBILEGLUES_VERSION_H
 
