diff --git a/src/gl/getter.cpp b/src/gl/getter.cpp
index 3e3dcb9..c5a0ad1 100644
--- a/src/gl/getter.cpp
+++ b/src/gl/getter.cpp
@@ -409,7 +409,7 @@ extern "C"
             return (const GLubyte*)rendererString.c_str();
         }
         case GL_SHADING_LANGUAGE_VERSION:
-            return (GLubyte*)"4.50 Krypton Wrapper with glslang and SPIRV-Cross";
+            return (GLubyte*)"4.50 Krypton Wrapper §f(§dYukari Launcher§f)§r";
         case GL_PROGRAM_ERROR_STRING_ARB:
             return (GLubyte*)glstate->glsl->error_msg;
         default:
diff --git a/src/gl/init.c b/src/gl/init.c
index c8fbd8e..475f1bc 100644
--- a/src/gl/init.c
+++ b/src/gl/init.c
@@ -436,13 +436,13 @@ void initialize_gl4es() {
     if (env_version) {
         snprintf(globals4es.version, 64,
                  globals4es.use_mc_color == 1 ? "%s §aKrypton Wrapper %s%s%d.%d.%d%s§r"
-                                              : "%s Krypton Wrapper %s%s%d.%d.%d%s",
+                                               : "%s Krypton Wrapper %s%s%d.%d.%d%s",
                  env_version, version_color, VERSION_TYPE, MAJOR, MINOR, REVISION, VERSION_SUFFIX);
         SHUT_LOGD("Targeting OpenGL %s", env_version);
     } else {
         snprintf(globals4es.version, 64,
                  globals4es.use_mc_color == 1 ? "%d.%d §aKrypton Wrapper %s%s%d.%d.%d%s§r"
-                                              : "%d.%d Krypton Wrapper %s%s%d.%d.%d%s",
+                                               : "%d.%d Krypton Wrapper %s%s%d.%d.%d%s",
                  globals4es.gl / 10, globals4es.gl % 10, version_color, VERSION_TYPE, MAJOR, MINOR, REVISION,
                  VERSION_SUFFIX);
         SHUT_LOGD("Targeting OpenGL %d.%d", globals4es.gl / 10, globals4es.gl % 10);
diff --git a/version.h b/version.h
index 94bcf3d..5cb5ec6 100644
--- a/version.h
+++ b/version.h
@@ -5,7 +5,7 @@
 #define MINOR 4
 #define REVISION 5
 #define VERSION_TYPE "Release "
-#define VERSION_SUFFIX ""
+#define VERSION_SUFFIX " §f(§dYukari Launcher§f)"
 #define RELEASE 1
 
 #endif //_GL4ES_VERSION_H
